minifi-toolkit/minifi-toolkit-configuration/src/test/java/org/apache/nifi/minifi/toolkit/configuration/dto/ProcessorSchemaTest.java

```diff
@@ -45,6 +45,35 @@
     private final String testKey = "testKey";
     private final String testValue = "testValue";
     private final String testPenalizationPeriod = "55 s";
+    private final String testAnnotationData = ">&lt;criteria&gt;\n" +
```

Comment:
Most certainly, just wanted to make sure I wasn't overlooking.  Will merge it in.